if require then
    local AllMutations: TableMutations = {}
    local MutationHandler = require(game:GetService("ReplicatedStorage").Modules.MutationHandler)
    for i, _ in next, MutationHandler.MutationNames do
        AllMutations[#AllMutations + 1] = i
    end
    return AllMutations
else
    return {
        "Eclipsed",
        "Meteoric",
        "Cosmic",
        "Cyclonic",
        "Fried",
        "Drenched",
        "Tempestuous",
        "Shocked",
        "Aurora",
        "Choc",
        "Bloodlit",
        "Spaghetti",
        "Sliced",
        "Maelstrom",
        "Verdant",
        "Glimmering",
        "Infected",
        "Enchanted",
        "Junkshock",
        "AncientAmber",
        "Twisted",
        "Alienlike",
        "FoxfireChakra",
        "CorruptChakra",
        "Cooked",
        "Chilled",
        "Heavenly",
        "Beanbound",
        "Fortune",
        "Aromatic",
        "Boil",
        "Celestial",
        "Warped",
        "Windstruck",
        "Burnt",
        "Subzero",
        "Wet",
        "Galactic",
        "Corrosive",
        "HoneyGlazed",
        "Sundried",
        "Blitzshock",
        "Rot",
        "Disco",
        "Friendbound",
        "Zombified",
        "Paradisal",
        "Jackpot",
        "Clay",
        "Amber",
        "Brainrot",
        "HarmonisedFoxfireChakra",
        "Frozen",
        "Corrupt",
        "Meatball",
        "Pollinated",
        "Dawnbound",
        "Molten",
        "Sandy",
        "Sauce",
        "Moonlit",
        "Oil",
        "Toxic",
        "Ceramic",
        "Enlightened",
        "Gnomed",
        "CorruptFoxfireChakra",
        "Stormcharged",
        "Static",
        "HarmonisedChakra",
        "Chakra",
        "Pasta",
        "Tranquil",
        "Plasma",
        "Touchdown",
        "Cloudtouched",
        "Radioactive",
        "Wiltproof",
        "Gloom",
        "Bloom",
        "Lightcycle",
        "OldAmber",
        "Voidtouched",
        "Glitched",
        "Acidic",
    }
end
